rmgpy/molecule/atomtype.py

```diff
@@ -91,6 +111,14 @@ def __init__(self, label='', generic=None, specific=None):
         self.decrementRadical = []
         self.incrementLonePair = []
         self.decrementLonePair = []
+        self.single = [] if single is None else single
+        self.allDouble = [] if allDouble is None else allDouble
+        self.rDouble = [] if rDouble is None else rDouble
+        self.oDouble = [] if oDouble is None else oDouble
+        self.sDouble = [] if sDouble is None else sDouble
+        self.triple = [] if triple is None else triple
+        self.benzene = [] if benzene is None else benzene
+        self.lonePairs = [] if lonePairs is None else lonePairs
```

Comment:
I think these can all be simplified to `single or []`, etc. similar to lines 104 and 105.